.prettierrc

```diff
@@ -0,0 +1 @@
+{ "semi": false, "singleQuote": true }
```

Comment:
Ahh here is the rc. Just add trailing ES5 trailing commas and I'm happy 👍

Reply:
Done and done